bitnami/redmine/README.md

```diff
@@ -556,6 +556,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 25.0.0
+
+This major release bumps the MariaDB version to 11.1. No major issues are expected during the upgrade.
```

Comment:
Could you please use this format?

This major release bumps the MariaDB version to 11.1. Follow the [upstream instructions](https://mariadb.com/kb/en/upgrading/) for upgrading to MariaDB 11.1. No major issues are expected during the upgrade.

Reply:
But there are no existing instructions 11.0 and 11.1. In the other versions exist instructions.

Reply:
That's why I suggest using the general link instead of the specific one.

Reply:
In other cases that is not added, why in this case is necessary?

Reply:
MariaDB always adds a link. You can check other README. Let's handle the unification of the Upgrade Notes in a different PR.